docs/mission-makers/olsen-gear.md

```diff
@@ -64,3 +64,9 @@ Place this in a folder with your gear scripts such as ``loadouts``.
 
 The path to your file would then be ``loadouts/GearDef.hpp``
 
+Next select the UO Framework button at the top of the editor and select "Configure Gear", followed by "Configure Gear Settings" where you need to check the "Enable Olsen System" checkbox.
+
+IMAGE HERE
+
+Now, simply add whatever gearscripts (.sqf files) you want in the folder alongside ``GearDef.hpp`` and you're ready to use you Olsen gearscripts via the UO Framework!
```

Comment:
In regards to my other comment it might would be a good idea to combine this sentence with theis one "Next, if your gear scripts are not going solely within GearScript.sqf, add you must add a file called GearDef.hpp and add this code to it;"

Example for sentence above the code line:
"In the case that you are using multiple gearscripts/multiple .sqf files you need to add a file called GearDef.hpp with the following code. Simply add all gearscripts/.sqf files you want to use alongside the "GearDef.hpp" into your mission folder"
(is mission folder correct or should this be in its own subfolder in the mission folder?)